app/Http/Controllers/ServiceAreaController.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\ServiceArea;
+use Illuminate\Http\Request;
+
+class ServiceAreaController extends Controller
+{
+    //
+    public function serviceAreaView(Request $request){
+        $requestContainer = array (
+            'zip_code' =>$request->zip_code
+        );
+
+        $zipCodeData = ServiceArea::serviceAreaZipCode($requestContainer);
+        
+        if(count($zipCodeData)>0){
+
+            
+            $schedule = $zipCodeData[0]->schedule;
+            $schedules = explode(",", $schedule);
+            $today = date ("D");
+            $todayUpper = strtoupper($today);
```

Comment:
you can actually do this. 
```suggestion
            $today = strtoupper(date("D"));
```